editor/src/components/canvas/canvas-strategies/strategies/grid-rearrange-keyboard-strategy.ts

```diff
@@ -133,6 +133,7 @@ export function gridRearrangeResizeKeyboardStrategy(
           gridRowStart,
           gridRowEnd,
         }),
+        'rerender-all-elements',
```

Comment:
possibly one to fix here too

Reply:
Added comment about follow up in 38ec355d5.